src/02-data-types.js

```diff
@@ -22,7 +24,9 @@ function getLengthOfString(str) {}
  *  convertToNumber("111");
  *  //> 111
  */
-function convertToNumber(val) {}
+function convertToNumber(val) {
+  return Number(val);
```

Comment:
Great job here! Using `Number` (rather than `parseFloat` or `parseInt` or `+`) adds a lot to the code's readability.